jetty-io/src/main/java/org/eclipse/jetty/io/MemoryPool.java

```diff
@@ -0,0 +1,21 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2021 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.io;
+
+public interface MemoryPool<T>
```

Comment:
javadoc needed.

Reply:
Added some.

Reply:
A bit minimal.  I think you need to say that this is a Pool of objects representing memory that can be acquired based on size.  The held instances may be the memory component itself (eg ByteBuffer) or an abstraction providing access to the memory component

Reply:
Thanks for the description, I improved the javadoc based on it.